tock/hours/models.py

```diff
@@ -237,6 +238,9 @@ class Timecard(models.Model):
                                             default=Decimal(settings.DEFAULT_BILLABLE_EXPECTATION), decimal_places=2, max_digits=3,
                                             verbose_name="Percentage of hours which are expected to be billable this week")
 
+    organization = models.ForeignKey(Organization, null=True, blank=True, on_delete=models.CASCADE)
```

Comment:
Wondering two things:
1. Should they be required?
2. Should they be non-editable, since they're auto-set?

Reply:
1. It makes sense to me for them to be required, but at this level our hands are somewhat tied by the fact that they're not required at the `User` level
2. Yes, I think so

Reply:
Ooo, I think these **need** to be editable so that timecards can be corrected via the admin when necessary.